src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Circle {
+}
```

Comment:
There is an unnecessary empty line at the beginning of the class definition. Please remove it to adhere to the checklist guidelines.